src/js/themes/hpe.js

```diff
@@ -14,12 +14,12 @@ import { Unsorted } from 'grommet-icons/icons/Unsorted';
 
 import { colors } from './colors';
 
-const isObject = (item) =>
+const isObject = item =>
```

Comment:
seems like we lost some prettier changes that we want

Reply:
fixing now